studio-docs/source/schema-checks.mdx

```diff
@@ -31,9 +31,9 @@ Try changing something in your schema and see what happens! If everything is set
 
 When you check your schema with the `apollo service:check` command:
 
-1. Apollo Studio generates a diff between your local schema and the schema active on the variant you are checking against (`current`, by default).
-2. It uses this diff to determine whether the changes affect any operations that have been executed against your graph recently (you can [customize this time window](#customizing-checks)).
-3. Studio returns the diff and a list of any breaking changes found.
+1. Apollo Studio generates a diff between your local schema and the active schema for the variant you're checking against (the default variant is `current`).
+2. Studio uses this diff to determine whether the changes affect any operations that have been executed against your graph recently (you can [customize this time window](#customizing-checks)).
+3. Studio returns the diff, along with a list of any breaking changes found.
```

Comment:
sorry i totally forgot to mention this but the new checks page is now operation centric, so maybe we should say "Studio returns the diff, along with a list of any affected operations found."